DESCRIPTION

```diff
@@ -8,7 +8,7 @@ Description: TODO
 License: MIT + file LICENSE
 Encoding: UTF-8
 Roxygen: list(markdown = TRUE)
-RoxygenNote: 7.1.2
+RoxygenNote: 7.2.0
 Depends: 
     R (>= 2.10)
 Imports:
```

Comment:
Sorry, one last thing:

Can you add gt to the Imports here? RCMD Check will fail otherwise.